CHANGELOG.md

```diff
@@ -1,7 +1,26 @@
-## next release
+## 2.3.0 / 2018-06-05
 
+* [FEATURE] Add query commands to promtool
+* [FEATURE] Add security headers to HTTP server responses
+* [FEATURE] Pass query hints via remote read API
 * [CHANGE] `marathon_sd`: use `auth_token` and `auth_token_file` for token-based authentication instead of `bearer_token` and `bearer_token_file` respectively.
+* [ENHANCEMENT] Optimise PromQL for memory usage and allocations
+* [ENHANCEMENT] Limit number of dropped targets in web UI
+* [ENHANCEMENT] Add advanced filterting configuration to Consul service discovery
+* [ENHANCEMENT] Add advanced filterting configuration to EC2 service discovery
 * [ENHANCEMENT] `marathon_sd`: adds support for basic and bearer authentication, plus all other common HTTP client options (TLS config, proxy URL, etc.)
+* [ENHANCEMENT] Provide machine type metadata and labels in GCE service discovery
+* [ENHANCEMENT] Use more efficient JSON encoder in API
```

Comment:
Put all the performance stuff together?

Reply:
You mean the JSON encoding and the query engine ones?
Arguably the JSON ones affect endpoints beyond querying.

Reply:
They do, but it might be nicer to try and put them together. I generally try to find some rough groupings when doing changelogs.

Reply:
Okay, done.